File: ModoEspejo.java
import java.util.Scanner;

public class ejercicio4 {
public static void main(String[] args) {
Scanner sc = new Scanner(System.in);
int [] numeros = new int[5];

    for(int i=0; i<numeros.length; i++){
        System.out.println("Ingrese el numero " + (i+1) + ": ");
        numeros[i]=sc.nextInt();
    }

    System.out.println("Arreglo original:");
    for(int i=0; i<numeros.length; i++){
        System.out.print(numeros [i] + " ");
    }
    System.out.println("");
    System.out.println("Arreglo espejo: ");
    for(int i=numeros.length-1; i>=0; i--){
        System.out.print(numeros[i] + " ");
    }
}
}

File: ModoEspejo.cpp
#include <iostream>>
using namespace std;

int main(){
    int numeros[5];

    for (int i = 0; i < 5; i++)
    {
        cout <<"Ingrese el numero " << i+1 << ": \n";
        cin >> numeros[i];
    }

    cout<<"Arreglo original \n";

    for (int i = 0; i < 5; i++)
    {
        cout <<numeros[i] <<" ";
    }
    
    cout<<"\nArreglo espejo \n";
    for(int i=4; i>=0; i--){
        cout << numeros[i] << " ";
    }
    return 0;
}